compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELogicalObjFifoSplittingUtils.h

```diff
@@ -11,11 +11,8 @@
 
 namespace mlir::iree_compiler::AMDAIE {
 
-/// Utility to help fetch those input DmaCpyNd Ops which needs to be split.
-SmallVector<AMDAIE::DmaCpyNdOp> fetchDmaCpyNdOpsToSplitOrCombine(Operation *op);
-
 /// Utility to split logicalobjectfifos given a vector of L2->L1 dma ops.
-LogicalResult splitLogicalObjectFifos(
+LogicalResult splitThirdInputLogicalObjectFifos(
```

Comment:
Seems like the fact that the object fifos are all third inputs in the core isn't used in this function, so I propose removing it from the function name

Reply:
Maybe the name is not accurate. But what it really means is it has the third input dma in `amdaie.core(in : [xx,  xx, xx], out : [xx]) `, e.g. https://github.com/nod-ai/iree-amd-aie/blob/main/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/split_logicalobjfifos_for_connection_reuse.mlir#L135 vs https://github.com/nod-ai/iree-amd-aie/blob/main/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/split_logicalobjfifos_for_connection_reuse.mlir#L36.

Reply:
Very simplistic example explaining my problem with the name:

```
def doubled_values_less_than_ten(X):
  return [2*v for v in X]
  
def main():
  vals = [1, 20]
  vals_less_than_ten = [x for x in vals if x < 10]
  return doubled_values_less_than_ten(vals_less_than_ten)
```

`double_values_less_than_ten` should be called `doubled_values`

i.e. where is the fact they're third arguments used, afaict it's not relevant.

Reply:
How about name "splitLogicalObjectFifosForElementwiseOp" or "splitLogicalObjectFifosForThirdInputInCore"?

Reply:
I think those names have the same issue -- where inside the function is it used that this comes from an elementwise op, or from the third arg in a core?  A function name should say just what it does, not who for :) Maybe this func shouldn't be exposed as a utility until we've generalized it more, will it be used outside of the original pass in a follow up PR? But anyway I know this isn't original code in PR, so I'm happy to accept it with any name and we can get back to a refactor / rename later, my vote of the 3 goes for splitLogicalObjectFifosForElementwiseOp